components/Staff/StaffMap.jsx

```diff
@@ -0,0 +1,21 @@
+const StaffMap = [
```

Comment:
you should make the file .tsx instead of .jsx. Using Typescript is a lot safer than using Javascript and a lot easier to debug because typescript is able to catch errors at runtime. We've been using .tsx (typescript) for everything so we should probably stick with that for this as well.

You will have to add whats called an interface that defines the types of the fields you created (name, title, image) you should look into it. Let me know if you have any questions!